cpp/include/kvikio/utils.hpp

```diff
@@ -308,6 +311,11 @@ struct libkvikio_domain {
       msg, nvtx3::payload { convert_to_64bit(val) }           \
     }                                                         \
   }
+#else
+#define KVIKIO_NVTX_FUNC_RANGE_2(msg, val) \
+  do {                                     \
+  } while (0)
+#endif
```

Comment:
Could shovel this into the shim, but because it only appears here that seems like overkill.